pkg/handler/iam_handler.go

```diff
@@ -139,11 +141,8 @@ func (h *IAMHandler) handlePolicy(ctx context.Context, p *v1alpha1.ResourcePolic
 			return fmt.Errorf("failed to get IAM policy: %w", err)
 		}
 
-		// TODO (#44): Continue to handle policy and alert updatePolicy error
-		// differently.
-		// Update the policy with new IAM binding additions.
 		if err := updatePolicy(cp, p.Bindings, expiry); err != nil {
-			return fmt.Errorf("failed to update IAM policy: %w", err)
+			logger.Warnf("encountered error while updating IAM policy for resource %q: %w", p.Resource, err)
```

Comment:
It doesn't seem we differentiate errors from adding new AOD bindings vs. removing expired AOD bindings? We should only ignore "remove expired AOD binding error".

Reply:
updating.